oak_functions_launcher/src/main.rs

```diff
@@ -19,7 +19,7 @@
 
 use anyhow::Context;
 use clap::Parser;
-use instance::{crosvm, native, LaunchedInstance};
+use instance::{native, virtualised, LaunchedInstance};
```

Comment:
```suggestion
use instance::{native, virtualized, LaunchedInstance};
```

Unfortunately I think we use american english everywhere else.